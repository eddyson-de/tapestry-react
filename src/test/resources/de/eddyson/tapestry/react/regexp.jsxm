export default (props)=>{
  props.text.split(/\n/).map((line)=><div>{line}</div>)
}