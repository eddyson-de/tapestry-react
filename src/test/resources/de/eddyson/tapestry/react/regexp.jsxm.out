define(["exports"], function (exports) {
  "use strict";

  Object.defineProperty(exports, "__esModule", {
    value: true
  });

  exports.default = function (props) {
    props.text.split(/\n/).map(function (line) {
      return React.createElement(
        "div",
        null,
        line
      );
    });
  };
});